website/templates/project/project.mako

```diff
@@ -292,7 +292,7 @@
                   }'></div>
             </div><!-- end containment -->
         % else:
-          <p>No components have been added to this project.</p>
+          <p>No components have been added to this ${node['category'].lower()}.</p>
```

Comment:
The text should display either "project" or "component". It should not show category names, e.g. "methods and materials". Is this the case? Note that there is Node property `project_or_component`.